pkg/controller/bypass/bypass_controller.go

```diff
@@ -103,16 +115,28 @@ func StartByPassController(client kubernetes.Interface) error {
 				log.Debugf("%s/%s: Pod is being deleted, skipping further processing", pod.GetNamespace(), pod.GetName())
 				return
 			}
+
+			log.Infof("%s/%s: DELETED", pod.GetNamespace(), pod.GetName())
```

Comment:
This message is misleading,  this pod is only being relabeld

Reply:
> This message is misleading, this pod is only being relabeld

I have modified it